site/docs/contributing.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Contributing to promptfoo
+sidebar_label: Contributing
+---
+
+**Note: This guide is a work in progress.**
+
+We welcome contributions from the community to help make promptfoo better. This guide will help you get started. If you have any questions, please reach out to us on [Discord](https://discord.gg/gHPS9jjfbs) or through a [GitHub issue](https://github.com/promptfoo/promptfoo/issues/new).
+
+## Project Overview
+
+promptfoo is an MIT licensed tool for testing and evaluating LLM apps. It allows you to:
+
+- Build reliable prompts, models, RAGs, and agents with benchmarks specific to your use case.
+- Speed up evaluations with caching, concurrency, and live reloading.
+- Score outputs automatically by defining metrics and perform automated red teaming.
+- Use as a CLI, library, or in CI/CD.
+- Use various LLM providers or integrate custom API providers.
+
+Our goal is to enable test-driven LLM development instead of trial-and-error.
```

Comment:
feel free to push back, but I think it's unnecessary to recap these details from the intro
```suggestion
promptfoo is an MIT licensed tool for testing and evaluating LLM apps.
```